core-web/apps/dotcms-ui/src/app/portlets/dot-categories/dot-categories-table/dot-categories-table.module.ts

```diff
@@ -0,0 +1,35 @@
+import { NgModule } from '@angular/core';
```

Comment:
As per the onboarding document in the portlet structure section, this module should be named: `DotCategoriesListing`, same for folder name.